db/migrate/20210501034609_add_legacy_lti_user_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLegacyLtiUserIdToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :legacy_lti_user_id, :string
```

Comment:
As you are doing a database lookup based on this, do you want an index on it?

Reply:
Yes